### PR TITLE
Configure `keyring` to use dbus secret service on linux

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
The migration from v2 to v3 forget to set the feature `linux-native-sync-persistent` for the `keyring` crate. This feature is required if we want the keyring to use the configured secret service on linux.

### CODE REVIEW

File: libparsec/crates/platform_device_loader/Cargo.toml [hunk 1]
```diff
@@ -25,7 +26,11 @@ log = { workspace = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 dirs = { workspace = true }
-keyring = { workspace = true, features = ["linux-native", "apple-native", "windows-native"] }
+keyring = { workspace = true, features = [
+    "linux-native-sync-persistent",
```
Comment (vxgmichel) [inline libparsec/crates/platform_device_loader/Cargo.toml:30]: Add comment
```
// Persistent means we use DBus's secret-services feature
```

File: libparsec/crates/platform_device_loader/Cargo.toml [hunk 2]
```diff
@@ -13,6 +13,7 @@ workspace = true
 
 [features]
 test-with-testbed = ["libparsec_testbed"]
+vendored-keyring = ["keyring/vendored"]
```
Comment (vxgmichel) [inline libparsec/crates/platform_device_loader/Cargo.toml:16]: this is not keyring this is vendored, but libdbus. So this feature should be called `vendored-dbus`

File: make.py [hunk 3]
```diff
@@ -38,20 +38,29 @@
 # distributed on pypi requires vendoring to comply with manylinux)
 # This is only needed for Linux as other OSs don't provide OpenSSL (and
 # hence it obviously must be vendored anytime we need it).
-MAYBE_FORCE_VENDORED_OPENSSL = ""
+maybe_force_vendored_libs_cargo_flags = maybe_force_vendored_openssl_cargo_flags = ""
```
Comment (vxgmichel) [inline make.py:41]: I would replace `maybe_force_vendored_libs_cargo_flags` by a `maybe_force_vendored_keyring_cargo_flags`

File: Cross.toml [hunk 4]
```diff
@@ -22,6 +22,7 @@ pre-build = [
 
 [target.x86_64-unknown-linux-musl.env]
 passthrough = [
+    "PKG_CONFIG_ALL_STATIC",
```
Comment (vxgmichel) [inline Cross.toml:25]: Add a small comment to explain what this is for

File: cli/Cargo.toml [hunk 5]
```diff
@@ -16,6 +16,7 @@ harness = false
 [features]
 use-sodiumoxide = ["libparsec_crypto/use-sodiumoxide"]
 vendored-openssl = ["libparsec_crypto/vendored-openssl"]
+vendored-keyring = ["libparsec/vendored-keyring"]
```
Comment (vxgmichel) [inline cli/Cargo.toml:19]: rename to `vendored-dbus`

File: libparsec/crates/platform_device_loader/Cargo.toml (hunk 1 above)
Comment (FirelightFlagboy) [reply libparsec/crates/platform_device_loader/Cargo.toml:30]: done

File: libparsec/crates/platform_device_loader/Cargo.toml (hunk 2 above)
Comment (FirelightFlagboy) [reply libparsec/crates/platform_device_loader/Cargo.toml:16]: done

File: cli/Cargo.toml (hunk 5 above)
Comment (FirelightFlagboy) [reply cli/Cargo.toml:19]: done

**Pull request merged**